wlauto/instrumentation/poller/__init__.py

```diff
@@ -0,0 +1,68 @@
+#    Copyright 2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=access-member-before-definition,attribute-defined-outside-init,unused-argument
+
+from wlauto import Instrument, Parameter, Executable
+from wlauto.exceptions import ConfigError
+from wlauto.utils.types import list_of_strs
+
+
+class Poller(Instrument)
+    name = 'poller'
+    description = """
+    Polls the given files at a set sample interval. The values are output in CSV format.
```

Comment:
It's worth elaborating in the following paragraphs -- how/where will the results appear (e.g. on per-iteration basis or in the top-level output)? What files can be specified? (e.g. does this assume that there will be a single integer or float in the file)? etc.